### PR TITLE
rename --key-file to --key on cli

### PR DESCRIPTION
**Pull request by arlimus**

### CODE REVIEW

File: bin/vulcano [hunk 1]
```diff
@@ -55,7 +55,7 @@ class VulcanoCLI < Thor
       desc: 'The login user for a remote scan.'
     option :password, type: :string, default: nil,
       desc: 'Login password for a remote scan, if required.'
-    option :key_file, type: :string, default: nil,
+    option :key, type: :string, default: nil,
```
Comment (chris-rock) [inline bin/vulcano:58]: i like it

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by chris-rock** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 2101f59e-fbad-4209-85a3-473a950b084a approved by: @chris-rock

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**